equinix/common_test.go

```diff
@@ -10,7 +10,7 @@ import (
 var (
 	preferable_plans  = []string{"x1.small.x86", "t1.small.x86", "c2.medium.x86", "c3.small.x86", "c3.medium.x86", "m3.small.x86"}
```

Comment:
this list should probably be trimmed down to just c3.small

Reply:
The intent of the list is to ensure that testing can find a matching combination of plan + os + metro. Matches earlier in the list should be prioritized (I don't recall from memory if that is the case)

Reply:
it can go in another PR.